apps/sms/views/shared/js/utils.js

```diff
@@ -942,6 +942,21 @@
       return promise.then(
         () => shimHostIframe.contentWindow.bootstrap(appInstanceId)
       );
+    },
+
+    onceDocumentIsVisible() {
```

Comment:
Yeah I think it's good timing to make this visibility promise into an independent util function.